client/zeth/core/mixer_client.py

```diff
@@ -515,16 +515,17 @@ def mix(
             self,
             mix_params: contracts.MixParameters,
             sender_eth_address: str,
-            pub_value: Optional[EtherValue] = None,
+            sender_eth_private_key: Optional[bytes],
```

Comment:
That's a general note, not related to this PR in particular.

As a general rule of thumb, passing secret keys/data around is not desirable (at all). That's something to be careful with (I comment here because I can see that we pass the private key to functions here)

Reply:
Sure. We can iterate in a future pass.